flink-core/src/main/java/org/apache/flink/configuration/RestartStrategyOptions.java

```diff
@@ -184,7 +184,7 @@ public class RestartStrategyOptions {
     public static final ConfigOption<Double> RESTART_STRATEGY_EXPONENTIAL_DELAY_BACKOFF_MULTIPLIER =
             ConfigOptions.key("restart-strategy.exponential-delay.backoff-multiplier")
                     .doubleType()
-                    .defaultValue(2.0)
+                    .defaultValue(1.2)
```

Comment:
Could we set this to `1.5` to make it a bit less aggressive?

Reply:
Thanks @mxm for the feedback!

`1.5` is fine for me, and I'd like to cc @zhuzhurk who propose change these 2 default value as well!

Max and Mason have a little feedback after voting in the [user mail list](https://lists.apache.org/thread/6glz0d57r8gtpzq4f71vf9066c5x6nyw), and Max and I had a offline discussion yesterday. Max think the 1.2 is a little small or aggressive. Here is the reason:

- Every time the job restarts, it will make a bunch of calls to the Kubernetes API, e.g. read/write to config maps, create task managers.
- When his producation had the default fixed-delay(1s) restart strategy turned on. A Kubernetes cluster became instable.

Following is the relationship between restart-attempts and retry-delay-time:

- The `delay-time` will reach 1 min after 12 attempts when `backoff-multiplier` is 1.5
- The `delay-time` will reach 1 min after 24 attempts when `backoff-multiplier` is 1.2

Hey @zhuzhurk , what do you think about setting`1.5` as the default value? If you agree it, I will update the FLIP and feedback it to the [user mail list](https://lists.apache.org/thread/6glz0d57r8gtpzq4f71vf9066c5x6nyw), and go ahead this PR.

![image](https://github.com/apache/flink/assets/38427477/642c57e0-b415-4326-af05-8b506c5fbb3a)

Reply:
1.5 sounds good to me. Yet I think we need to send the updates to the FLIP discussion ML and open another vote for it. In case it turned out to be a surprise for those who have reviewed and voted on the FLIP.

Reply:
Thanks @zhuzhurk for the quick feedback!

I have updated it in the `dev&user` mail list. Let us wait for more feedback for a while. Also, I pause FLINK-33736 first. Therefore, I will remove the first commit from this PR and continue working on the rest of the JIRA for FLIP-364.

[1] https://lists.apache.org/thread/6glz0d57r8gtpzq4f71vf9066c5x6nyw

Reply:
The default value related commit is migrated to a new [separate PR](https://github.com/apache/flink/pull/23911), we can follow it there, thanks~

Also, if this PR(FLINK-32895) no other comments in 72 hours, I will merge it, thanks.

Reply:
Thanks for considering our suggestion!